src/Powercord/plugins/translate.js

```diff
@@ -57,7 +76,8 @@ module.exports = class Translate extends Plugin {
                 .map(from => ({
                   type: 'button',
                   name: translate.languages[from],
-                  onClick: () => setText({ to, from })
+                  onClick: () => setText({ to,
```

Comment:
may be single lined?

Reply:
just ran eslint over it, and eslint made it two lined

Reply:
oh, fine then